example/linux/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Is `example/linux/` entirely generated by Flutter tool, without manual changes?

Reply:
The manual changes are:

- Build `libwebcrypto.so` in `linux/CMakeLists.txt`
- Add it to `webcrypto_bundled_libraries` so that it's getting included in the app bundle

You can test all of this by running the example with `flutter run -d linux`.